src/Cli.Tests/AddEntityTests.cs

```diff
@@ -1,21 +1,28 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Cli.Commands;
+
 namespace Cli.Tests
 {
     /// <summary>
     /// Tests for Adding new Entity.
     /// </summary>
     [TestClass]
     public class AddEntityTests
+        : VerifyBase
     {
-        /// <summary>
-        /// Setup the logger for CLI
-        /// </summary>
-        [ClassInitialize]
-        public static void SetupLoggerForCLI(TestContext context)
+        [TestInitialize]
```

Comment:
Any benifit of converting it to TestInitialize from ClassInitialize

Reply:
Ideally we want this set up for each test rather than a shared instance across all tests as that reduces the amount of cleanup we have to do when a test completes, as the objects are deallocated and the GC will get rid of them.

Reply:
I would imagine the performance delta is marginal. Having a filesystem (in memory) per test would prevent side effects of tests running concurrently, if in the future schema files are changed per-test.

Reply:
Yes, you're correct that it would increase memory needed but I'd argue the value of having truly isolated tests outweighs the impact that it has.